packages/mui-material/src/Alert/Alert.js

```diff
@@ -169,6 +182,7 @@ const Alert = React.forwardRef(function Alert(inProps, ref) {
     color,
     severity,
     variant,
+    colorSeverity: color || severity,
```

Comment:
This is required because we can't do:

```js
...Object.entries(theme.palette)
    .filter(([, value]) => value.main)
    .map(([color]) => ({
      // this will throw error because a function cannot read variable `color`.
      props: ({ ownerState }) => color === (ownerState.color || ownerState.severity) && ownerState.variant === 'outlined',
      style: {
        color: theme.vars
          ? theme.vars.palette.Alert[`${color}Color`]
          : getColor(theme.palette[color].light, 0.6),
        border: `1px solid ${(theme.vars || theme).palette[color].light}`,
        [`& .${alertClasses.icon}`]: theme.vars
          ? { color: theme.vars.palette.Alert[`${color}IconColor`] }
          : {
              color: theme.palette[color].main,
            },
      },
    })),
```